components/org.wso2.bfsi.consent.management.common/src/main/java/org/wso2/bfsi/consent/management/common/exceptions/ConsentManagementRuntimeException.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.bfsi.consent.management.common.exceptions;
+
+/**
+ * Used for runtime exceptions in consent management component.
+ */
+public class ConsentManagementRuntimeException extends RuntimeException {
+
+    public ConsentManagementRuntimeException(String message, Throwable e) {
+
+        super(message, e);
+    }
+
+    public ConsentManagementRuntimeException(String message) {
+        super(message);
+    }
```

Comment:
```suggestion
    public ConsentManagementRuntimeException(String message) {
    
        super(message);
    }
```
Let's add a new line after method signature. Let's keep it consistent checking the other places also.

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/1/commits/aa7e22c798e360153510f1565d76a7f7c81a4965